### DISCUSSION TITLE
Can tasks run 'remove duplicates' ?

### DISCUSSION THREAD

**Discussion by ninmonkey** [NickeManarin/ScreenToGif#1001] [Q&A]
I can't seem to find much about tasks. I found some options in the preferences dialog. Can I make it run remove duplicates, sum time,  at 100% similarity? I record a lot of command line gifs, so I frequently use that transform.

If it it's possible/easier with the command line, that could work

![image](https://user-images.githubusercontent.com/3892031/139864802-7447d55e-1958-4d54-bace-23e512c3620a.png)

**Comment by NickeManarin** [comment]
Not right now. 
The feature needs to be implemented first.

**Comment by yogiraturimansyah** [comment]
kaya mana selanjutnya untuk meneruskan akun cyber
